.github/workflows/csharp.yml

```diff
@@ -138,14 +96,23 @@ jobs:
           dotnet-version: 3.1.x
       - name: Restore dependencies
         run: dotnet restore OptimizelySDK.NetStandard20/OptimizelySDK.NetStandard20.csproj
-      - name: Build and sign Standard 2.0 project
-        id: netStandard20_build
+      - name: Build & strongly name assemblies
         run: dotnet build OptimizelySDK.NetStandard20/OptimizelySDK.NetStandard20.csproj /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=D:\a\csharp-sdk\csharp-sdk\keypair.snk -c Release
-      - name: Check on success
-        if: steps.netStandard20_build.outcome == 'success' && env.CURRENT_BRANCH == env.RELEASE_BRANCH && env.EVENT_TYPE == 'push'
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.OFTA_KEY }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.OFTA_SECRET }}
-          AWS_DEFAULT_REGION: ${{ secrets.OFTA_REGION }}
-        run: |
-          (aws s3 cp ./OptimizelySDK.NetStandard20/bin/Release/netstandard2.0/OptimizelySDK.NetStandard20.dll s3://optly-fs-travisci-artifacts/${{ env.REPO_SLUG }}/${{ env.BUILD_NUMBER }}/${{ env.RUN_NUMBER }}/${{ env.ATTEMPT_NUM }}/OptimizelySDK.NetStandard20.dll-unsigned)
+  
+  integration_tests:
+    name: Run Integration Tests
+    needs: [ netFrameworksAndUnitTest, netStandard16, netStandard20 ]
+    uses: optimizely/csharp-sdk/.github/workflows/integration_test.yml@master
+    secrets:
+      CI_USER_TOKEN: ${{ secrets.CI_USER_TOKEN }}
+      TRAVIS_COM_TOKEN: ${{ secrets.TRAVIS_COM_TOKEN }}
```

Comment:
Can't we remove all this "TRAVIS" terms? I believe, we do not have any more dependencies onto TRAVIS.

Reply:
Wondering why we can remove aws s3 for github actions. Can we add why in the PR summary for record?

Reply:
sounds good. we can remove all "TRAVIS" in a separate task.